src/librustc/session/config.rs

```diff
@@ -1532,6 +1532,15 @@ pub fn default_configuration(sess: &Session) -> ast::CrateConfig {
             }
         }
     }
+    if let Some(s) = &sess.opts.debugging_opts.sanitizer {
+        let s = match s {
```

Comment:
This could be a method on `Sanitizer`

Reply:
Replaced with display trait implementation.